org/mozilla/jss/ssl/javax/JSSEngine.java

```diff
@@ -804,6 +825,31 @@ public boolean isOutboundDone() {
      */
     public abstract SecurityStatusResult getStatus();
 
+    /**
+     * Gets the default configuration.
+     */
+    public HashMap<Integer, Integer> getDefaultConfiguration() {
+        HashMap<Integer, Integer> result = new HashMap<Integer, Integer>();
+        result.put(SSL.ENABLE_RENEGOTIATION, SSL.RENEGOTIATE_REQUIRES_XTN);
+        result.put(SSL.REQUIRE_SAFE_NEGOTIATION, 1);
+        result.put(SSL.ENABLE_FALLBACK_SCSV, 1);
+        return result;
+    }
+
+    /**
+     * Updates the configuration with the given value.
+     */
+    public void addConfiguration(int key, int value) {
```

Comment:
Should we call it `setConfiguration()` instead?

Reply:
Ah I was going to add a `setConfiguration(HashMap<...> config)` function, thanks!